cache-config/testing/docker/docker-compose.yml

```diff
@@ -82,7 +85,8 @@ services:
       - variables.env
     build:
       args:
-        RHEL_VERSION: ${RHEL_VERSION:-8}
+        - OS_DISTRO=${OS_DISTRO:-rockylinux/rockylinux}
```

Comment:
Rocky Linux now has an official image, `rockylinux/rockylinux` can be just `rockylinux`.